refinerycms-search.gemspec

```diff
@@ -8,6 +8,7 @@ Gem::Specification.new do |s|
   s.homepage          = %q{http://refinerycms.com}
   s.email             = %q{info@refinerycms.com}
   s.authors           = ["Philip Arndt", "Uģis Ozols", "Joe Sak"]
+  s.metadata          = { "contributor" => "Brice Sanchez <https://github.com/bricesanchez>" }
```

Comment:
No, it's because you've done a great job in the CMS that i'm able to improve this extension, i'm just a contributor ;)